src/utils/helpers/index.ts

```diff
@@ -1,7 +1,17 @@
+import { getOrCreateVaultAsAsset, getOrCreatePriceHistory } from "./yVault/yVault";
+import { BigDecimal, BigInt, Address, Bytes } from "@graphprotocol/graph-ts";
+
+
 export {
   getOrCreateVaultAsAsset,
 } from "./yVault/yVault";
 
 export {
   getOrCreatePriceHistory
 } from "./yVault/yVault";
+
+
+export function createPriceHistory(vaultAddress: Address, vaultName: String, totalSupply:BigInt, pricePerFullShare:BigInt, timestamp: BigInt, txnHash: Bytes): void {
```

Comment:
Good idea extracting this out here. To prevent duplicate code, delete this function from src/mappings/yVault.ts